packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -0,0 +1,248 @@
+import { IComboBoxStyles } from './ComboBox.Props';
+import {
+  ITheme,
+  IStyle,
+  mergeStyleSets,
+  FontSizes,
+  FontWeights,
+  getFocusStyle
+} from '../../Styling';
+import { memoizeFunction } from '../../Utilities';
+
+const MS_HIGHCONTRAST_ACTIVE = '@media screen and (-ms-high-contrast: active)';
+const MS_HIGHCONTRAST_BLACK_ON_WHITE = '@media screen and (-ms-high-contrast: black-on-white)';
+
+const ComboBoxHeight = '32px';
+const ComboBoxLineHeight = '30px';
+const ComboxBoxCaretDownWidth = '32px';
+const ComboBoxoptionHeight = '36px';
+
+const getDisabledStyles = (theme: ITheme): IStyle => {
+  const { semanticColors, palette } = theme;
+  return {
+    backgroundColor: semanticColors.disabledBackground,
+    borderColor: semanticColors.disabledBackground,
+    color: semanticColors.disabledText,
+    cursor: 'default',
+    [MS_HIGHCONTRAST_ACTIVE]: {
+      borderColor: palette.contrastBlackDisabled,
+      color: palette.contrastBlackDisabled
+    },
+
+    [MS_HIGHCONTRAST_BLACK_ON_WHITE]: {
+      borderColor: palette.contrastWhiteDisabled,
+      color: palette.contrastWhiteDisabled
+    },
+  };
+};
+
+const getListoptionHighContrastStyles = (theme: ITheme): IStyle => {
+  const { semanticColors, palette } = theme;
+  return {
+    [MS_HIGHCONTRAST_ACTIVE]: {
+      backgroundColor: palette.contrastBlackSelected,
+      borderColor: palette.contrastBlackSelected,
+      color: palette.black,

```

Comment:
see my comment on high contrast below
in addition to that, this will not work, because palette.black is themed, and could be white on a dark theme (the name is misleading)